esmvaltool/diag_scripts/zmnam/zmnam_clean.py

```diff
@@ -0,0 +1,14 @@
+"""
+Zonal-mean annular mode cleaning routine.
+
+Author: Federico Serva (ISAC-CNR & ISMAR-CNR, Italy)
+Copernicus C3S 34a lot 2 (MAGIC)
+"""
+
+import subprocess
+
+
+def zmnam_clean():
+    """Dispose of temporary files."""
+    subprocess.check_call('rm tmp_gh_da_an_zm_hem.nc', shell=True)
```

Comment:
```suggestion
    os.remove('rm tmp_gh_da_an_zm_hem.nc')
```

Reply:
Though note that there would probably be no need for this function anymore if the Python cdo package is used, as that takes care of cleaning up temporary files automatically.